mantis-control-plane/mantis-control-plane-server/src/main/java/io/mantisrx/master/api/akka/MasterApiAkkaService.java

```diff
@@ -209,18 +208,22 @@ private void startAPIServer() {
 
         ServerSettings customServerSettings = defaultSettings.withWebsocketSettings(customWebsocketSettings);
 
-        final CompletionStage<ServerBinding> binding = http.bindAndHandle(routeFlow,
-            ConnectHttp.toHost("0.0.0.0", port),
-            customServerSettings,
-            system.log(),
-            materializer);
-        binding.exceptionally(failure -> {
-            System.err.println("API service exited, committing suicide !" + failure.getMessage());
-            logger.info("Master API service exited in error, committing suicide !");
-            system.terminate();
-            System.exit(2);
-            return null;
-        });
+        final CompletionStage<ServerBinding> binding = Http.get(system)
+                .newServerAt("0.0.0.0", port)
+                .withSettings(customServerSettings)
+                .connectionSource()
+                .to(Sink.foreach(connection -> {
+                    MasterApiMetrics.getInstance().incrementIncomingRequestCount();
+                    connection.handleWith(routeFlow, materializer);
+                }))
+                .run(materializer)
+                .exceptionally(failure -> {
+                    System.err.println("API service exited, committing suicide !" + failure.getMessage());
+                    logger.info("Master API service exited in error, committing suicide !");
+                    system.terminate();
+                    System.exit(2);
+                    return null;
+                });
         logger.info("Starting Mantis Master API on port {}", port);
```

Comment:
What is this for?

Reply:
Remove deprecated akka api + log incrementIncomingRequestCount.